src/server/utilities/liteATIClickTracking/index.test.ts

```diff
@@ -0,0 +1,118 @@
+import trackingScript from '.';
+
+const dispatchClick = (targetElement: HTMLElement) => {
+  document.body.appendChild(targetElement);
+  const event = new MouseEvent('click', {
+    view: window,
+    bubbles: true,
+    cancelable: true,
+  });
+  targetElement.dispatchEvent(event);
+};
+
+describe('Click tracking script', () => {
+  const randomUUIDMock = jest.fn();
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.useFakeTimers().setSystemTime(new Date(1731515402000));
+
+    trackingScript();
+
+    document.cookie =
+      'atuserid=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/;';
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    delete window.location;
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    window.location = {
+      assign: jest.fn(),
+    };
+
+    window.sendBeaconLite = jest.fn();
+
+    Object.defineProperty(global, 'crypto', {
+      value: {
+        randomUUID: randomUUIDMock,
+      },
+    });
```

Comment:
Not for spike but we should probably move this to the src/testHelpers/jest-setup.js file so that we don't have to keep mocking it in other places